src/roster/views.py

```diff
@@ -1475,6 +1475,7 @@ def __init__(self):
 
     def get(self, request, *args, **kwargs):
         self.clusive_user = request.clusive_user
+        self.days = kwargs['days']
```

Comment:
We need a default value for `days` here because one of the endpoints is `/details/<username>` with no time scale specified.

Regarding defaults, @bgoldowsky suggested in the JIRA that "We can use the existing ClusiveUser field as the persisted default choice, so that whichever time period you have picked on the dashboard carries over to this page (may want to discuss if this is the desired UX)"

I agree with @bgoldowsky 's suggestion; intuitively, if the teacher has intentionally set a time scale on the dashboard, they probably expect the same time scale on the details page.  I would go with that until UX says otherwise.

Also, @cindyli suggested the other day (private conversation) that the student listed on the Manage page could also be links to their details page.  Currently they are just static text.  I agree with @cindyli's suggestion, but there is no time scale associated with the Manage page.  Overall, if we continue to support the `/details/<username>` endpoint, we need some default time scale.  The Dashboard defaults to 7 days (I think), so we could do the same here:
```
self.days = kwargs.get('days', 7)
```

Reply:
Since the requirement so far to land on the student details page is from the dashboard "Active reading time" panel where a time is always selected and will be carried to the student details page. I've removed the support of `/details/<username>`.

Reply:
👍 
I thought about it a bit more.

The student links on the Dashboard are inside the "Active reading time".  That context implies that the user is requesting more details about that student and reading information.

Having links to student details on the Manage page lacks that context so it's not obvious where the links go in that case.  Also, the Manage page is about managing the period -- modifying the roster, modifying individual student account information, and the name of the period.  That seems distant for the kind of information a student details link would provide.

For that reason, I no longer think that details links are useful on the Manage page.  But, in any case, I have asked the UX team for their opinion.